AppCenter-Windows.sln

```diff
@@ -205,8 +205,8 @@ Global
 		{3FE04B97-48DE-4895-8612-ECBDFEBD917C}.Release|x64.Build.0 = Release|Any CPU
 		{3FE04B97-48DE-4895-8612-ECBDFEBD917C}.Release|x86.ActiveCfg = Release|Any CPU
 		{3FE04B97-48DE-4895-8612-ECBDFEBD917C}.Release|x86.Build.0 = Release|Any CPU
-		{15E3FC9B-7714-4468-A296-81627A5B8F01}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{15E3FC9B-7714-4468-A296-81627A5B8F01}.Debug|Any CPU.Build.0 = Debug|x86
+		{15E3FC9B-7714-4468-A296-81627A5B8F01}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Which project is this?

Reply:
`Microsoft.AppCenter.UWP`

Reply:
Closing conversation for now. Reopen if necessary

Reply:
What are those changes for in that file? @Jamminroot

Reply:
Followed up offline.